docs/CONNECTOR_PROTOCOL.md

```diff
@@ -127,6 +127,7 @@ In addition to the connector index `.elastic-connectors`, we have an additional
   error: string; -> Optional error message
   index_name: string; -> The name of the content index
   worker_hostname: string; -> The hostname of the worker to run the sync job
+  ingestion_type: string; -> Enum: [full, partial]
```

Comment:
My main question here is "what does ingestion_type actually means?".

Is it something that does not have any meaning and only the connector itself gives meaning to it? (e.g. connector defines "full", "incremental" and "apples" syncs and user can just put a different schedule for them, but the connector itself does the different handling of different types by itself)

Or is it something that also modifies the behaviour of the framework? (e.g. `partial` = only upserts, `full` = upserts + deletes).